configuration/scripts/options/set_nml.alt03

```diff
@@ -23,3 +23,4 @@ Ktens          = 0.
 e_ratio        = 2.
 seabed_stress    = .true.
 use_bathymetry = .true.
+l_mpond_fresh = .true.
```

Comment:
If particular changes are addressing lines in the spreadsheet, could you add a note on the line that says which test script was changed, and how?

Reply:
This recommendation came from #437.  https://github.com/CICE-Consortium/CICE/issues/437#issuecomment-734004147.  But it doesn't change answers which is a little concerning and something that we need to look into more probably.

Reply:
Here's what to look for:  l_mpond_fresh might only be used for coupling, and the mixed layer model only calculates heat fluxes, not water/salt, so  l_mpond_fresh might not be testable in our standalone configurations.

Reply:
So far as I can tell, l_mpond_fresh shows up in CICE_RunMod.F90 in the standalone model as

```
            if (l_mpond_fresh) then
               fpond(i,j,iblk) = fpond(i,j,iblk) * rhofresh/dt
               fresh(i,j,iblk) = fresh(i,j,iblk) - fpond(i,j,iblk)
            endif
```

I'm not sure what happens to fpond or fresh after that, but it doesn't seem to impact the solution.  I guess I think that's OK.  When we turn on options, it would be nice to see an impact, but if that's not possible, I still think it worthwhile to have a test that turns the flag on and at least tests the code in a technical sense.